### ISSUE TITLE
tests don't work on mac os catalina

### ISSUE THREAD

**Issue by christophsturm** [JetBrains/Exposed#894]
running the test suite fails with this stack trace 
```
org.jetbrains.exposed.DefaultsTest > testDefaultsWithExplicit01 STANDARD_ERROR
    Exception in thread "driver_launched_mysqld_1" com.mysql.management.util.WrappedException: java.io.IOException: Cannot run program "/private/var/folders/99/084s2g6s4pbgwr4_46ysxms80000gn/T/test-c.mxj/mysql-c.mxj/bin/mysqld": error=86, Bad CPU type in executable
        at com.mysql.management.util.Exceptions.toRuntime(Exceptions.java:55)
        at com.mysql.management.util.Exceptions$Block.exec(Exceptions.java:89)
        at com.mysql.management.util.RuntimeI$Default.exec(RuntimeI.java:180)
        at com.mysql.management.util.Shell$Default.run(Shell.java:140)
        at java.base/java.lang.Thread.run(Thread.java:835)
```

the reason is proably that catalina dropped support for 32 bit executables. 

this is also mentioned here: https://github.com/JetBrains/Exposed/issues/804

**Comment by christophsturm** [comment]
the problem probably lies in mysql-connector-mxj, and is not so easy to fix because mysql-connector-mxj had its latest release in 2011

**Comment by christophsturm** [comment]
related: https://stackoverflow.com/questions/59086476/mysqldresource-isnt-working-with-macos-catalina

**Comment by KushalP** [comment]
This can now be closed because #957 is merged.
